A/Argp/build_tarballs.jl

```diff
@@ -0,0 +1,44 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Argp"
+version = v"1.4.1"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/ericonr/argp-standalone.git", "743004c68e7358fb9cd4737450f2d9a34076aadf"),
+    DirectorySource("./bundled")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd ${WORKSPACE}/srcdir/argp-standalone
+
+for f in ${WORKSPACE}/srcdir/patches/*.patch; do
+    atomic_patch -p1 ${f}
+done
+
+autoreconf -i
+./configure --prefix=$prefix --build=${MACHTYPE} --host=${target}
+make CC=gcc
```

Comment:
```suggestion
make CC=${CC}
```
Don't hardcode GCC. We prefer to use Clang on some platforms where that is the better supported compiler (e.g. macOS).

Reply:
Well, I had initially let the system decide, and compilation breaks with clang. That's why I forced it to gcc.

Reply:
What were the errors?